Dockerfile

```diff
@@ -45,7 +45,7 @@ RUN apt-get update && apt-get install -y \
     vim
 
 # SYNC SCRIPT CREATION AND SCHEDULING
-RUN echo "#!/bin/sh\nwhile true; do\nrsync -a /home/$username/ /data; sleep 5\ndone" > /data-sync.sh && \
+RUN echo "#!/bin/sh\nrsync -a /home/$username/ /data\n" > /data-sync.sh && \
```

Comment:
Can we have this run on a cron every ~15m? I'm still concerned about an uncontrolled shutdown of the host resulting in lost data.

Reply:
oh for sure, first pass was just syncing on quit

Reply:
![I'll allow it](https://media.giphy.com/media/146heXDX89mUgw/giphy.gif)